module3/projects/consultancy/index.md

```diff
@@ -0,0 +1,88 @@
+
+### Feedback from Previous students:
+
+  * "The complexity added value to the interviewing conversations. I got to speak to pros/cons, and I understood the concepts a lot better."
+  * "Exploring different design patterns was great and being able to speak on different ways to build things PLUS being able to compare was helpful."
+  * "Incredibly helpful in interviewing. It was the biggest project on my resume, and was key to my early interviews. It was more important in my interview for my new role than my Mod 4 capstone because of the multi-app architecture. I could speak to decision-making more because I got to work on each piece. My new company said I reset their expectations for what a bootcamp grad could accomplish."
+  * "It came up in just about every interview I had."
+  * "People seemed most excited/interested about working on a larger team project. They asked lots of followup questions about how we handled project management, made decisions, etc. It was really helpful to know how SOA/multi-app systems worked."
+  * "They were surprised that we got exposure to SOA and speaking about microservices.
+  It helped to understand and speak to Abstraction and Encapsulation a lot more. Working o
+  n a larger team definitely helped me get started at my job which was a way bigger team t
+  han the 10-person project."
+  * "People seemed most excited/interested about working on a larger team project (10 p
+  eople). They asked lots of followup questions about how we handled project management, m
+  ade decisions, etc. It was really helpful to know how SOA/multi-app systems worked."
+  * "It definitely came up in interviews, especially about microservices, and the trade
+  offs versus monolith apps."
+
+### User Empathy, Personas, and Equity Analysis
+
+We'll also examine our end users on a deeper level to instill a strong sense of developer empathy. We will do this by focusing time on "user personas" to answer questions like "who is our user", "what age range are we targeting", "how tech-savvy must the user be to use our app" and so on. As part of this project, groups will also perform an equity analysis in order to interrogate the product they're building, and reflect on possible changes that could better prioritize inclusion and harm reduction. 
+
+## Phases
+
+* [Ideation](./ideation)
+* [Inception](./inception)
+* [Development](./development)
+* [Retro](./retro_guide)
+* [Presentation](./presentation)
+* [Evaluation](./evaluation)
+
+## Technical Requirements
+
+You have a lot of freedom with this project, but there are a few technical requirements. Overall, your project must:
+
+1. Includes at least one Rails API and one ReactJS application.
+2. Use one external API, but it can be consumed on the FE or BE. 
+3. Be deployed
```

Comment:
Do we want the frontend deployed? We might want to include some helpful hints for that since deploying React apps can be a little more complex than just using GH Pages

Reply:
Yes I think we should. What do you think about requiring deployment for their pair project in M3 so they have a bit more practice before consultancy?

Reply:
If that's too much, we can add some deployment guidelines here. Do any exist already on the current FE site?

Reply:
I added a note to the FE requirements about researching and choosing a deployment platform for the FE application.  Done.